.travis.yml

```diff
@@ -1,12 +1,11 @@
 language: ruby
 cache: bundler
 rvm:
-  - 2.1.10
-  - 2.2.10
   - 2.3.8
   - 2.4.7
   - 2.5.6
   - 2.6.4
```

Comment:
Optional: What about bumping all those Ruby versions to the latest?

It would be: 2.4.9, 2.5.7, 2.6.5